### PR TITLE
Sheduler for odoo_entity daily update

### PR DESCRIPTION
**Pull request by Vlad-CL**

### CODE REVIEW

File: clx_utils/models/clx_mysql.py [hunk 1]
```diff
@@ -109,3 +109,24 @@ def _read_db_config(self):
         }
 
         return db
+
+    def update_odoo_entity_table(self):
```
Comment (toddcd) [inline clx_utils/models/clx_mysql.py:113]: Curious when we're calling this?

**Review by toddcd — APPROVED** [review]

File: clx_utils/models/clx_mysql.py (hunk 1 above)
Comment (Vlad-CL) [reply clx_utils/models/clx_mysql.py:113]: From the scheduler.
image

Comment (toddcd) [reply clx_utils/models/clx_mysql.py:113]: I see where we're calling this. I'm curious when, how often, etc...? I've not worked with the Odoo scheduler and when looking at your code it's not obvious when this script is getting run.

Comment (toddcd) [reply clx_utils/models/clx_mysql.py:113]: looks like once a day?

**Pull request merged**